libphal/libphal.H

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+extern "C" {
+#include <libpdbg.h>
+}
+
+namespace libphal
+{
+namespace sbe
+{
+
+/**
+ * @brief utility function to check SBE state to allow chipops.
+ *
+ * Note: Caller has to make sure pib belong to functional/dump functional
+ *       processor
+ *
+ * @param[in] target pib target to operate on
+ *
+ * Exceptions: SbeError
+ */
+void validateSBEState(struct pdbg_target *pib);
+
+/**
+ * @brief utility function to capture SBE ffdc data using
+ *        pdbg function sbe_ffdc_get().
+ *        For regular chip-op failure, SBE FFDC response data copied
+ *        into a temporary file.
+ *
+ * Exceptions: SbeError with failure reason code, File name and fd.
+ *             std::runtime_error - for file operation failure.
+ */
+void captureFFDC(struct pdbg_target *pib);
```

Comment:
const

Reply:
Just relaized pdbg back-end api's used non const. soe we should use non const